flight/PiOS/posix/pios_sys.c

```diff
@@ -639,13 +639,15 @@ void PIOS_SYS_Args(int argc, char *argv[]) {
 				hw_argseen = false;
 				break;
 			}
+#if !(defined(_WIN32) || defined(WIN32) || defined(__MINGW32__))
```

Comment:
Need to do the same in usage.

Reply:
Right now I don't do *anything* to make usage correct / limit to what's applicable :/

Reply:
Oh I guess I do a little.  Will look.  Usually not worth blocking ab uild fix for something like this

Reply:
OK, it's done, but not really correct still (top line of summary doesn't change).